test/windows/acceptance/logstash_release_acceptance.ps1

```diff
@@ -0,0 +1,71 @@
+# Created By: Gabriel Moskovicz
+#
+# To be run on Jenkins
+#
+# Requirements to run the test:
+# 
+# - Powershell 4
+# - Windows 7 or newer
+# - Java 7 or newer
+
+$LS_CONFIG="test.conf"
+$LS_BRANCH="1.5"
+$Logstash_path = "C:\logstash"
+$Logstash_Snapshot_Directory = "$Logstash_path\logstash-latest-SNAPSHOT.zip"
+$Logstas_URL = "https://s3-eu-west-1.amazonaws.com/build-eu.elasticsearch.org/logstash/$LS_BRANCH/nightly/JDK7/logstash-latest-SNAPSHOT.zip"
+
+If (Test-Path $Logstash_path){
+	ri -Recurse -Force $Logstash_path
+}
+
+md -Path $Logstash_path
+(New-Object System.Net.WebClient).DownloadFile($Logstas_URL, $Logstash_Snapshot_Directory)
+
+#Unzip file
+$Destination = "$Logstash_path\logstash_" + $LS_BRANCH
+Add-Type -assembly "system.io.compression.filesystem"
+[io.compression.zipfile]::ExtractToDirectory($Logstash_Snapshot_Directory, $Destination)
+
+#Remove old files
+ri $Logstash_Snapshot_Directory
+
+#Move folder
+cd $Destination
+mv log* logstash
+cd logstash
+
+#Create Configuration
+ni $LS_CONFIG -it file
+sc -Path $LS_CONFIG -Encoding ascii -Value "input { 
+	tcp { 
+			port => 2000 
```

Comment:
I am saying this because we had issue in the past when the port was already used by another test.